src/chocolatey-core.extension/README.md

```diff
@@ -1,6 +1,15 @@
 # chocolatey-core.extension
 
-This is the Powershell module that extends Chocolatey with new functions.
+This is a extension package that provides helper functions installed as a Chocolatey extension.
+These functions may be used in Chocolatey install/uninstall scripts by declaring this package a dependency in your package's nuspec.
+This package provides helper functions that can be used to enhance what is already available out of the box with Chocolatey CLI.
+This includes both features that are being tested for Chocolatey CLI itself, and other helpers to make the maintenance of
+Chocolatey packages easier.
+
+Helpers that was available in this package, and laten been added to Chocolatey CLI they will be moved to
```

Comment:
Didn't see this one on first pass, sorry, looks like a copy of the the other one.

```suggestion
Helpers that were available in this package, and were later added to Chocolatey CLI, will be moved to
```